### PR TITLE
Upgrade Corda to 4.8

### PR DESCRIPTION
**Pull request by sanvenDev**
1. Upgrade CorDapps to Corda 4.8.
2. Upgrade corda network to Corda 4.8.
3. Add partyB to the Corda network.
4. Update the certificates of corda network (because version change required to generate new credentials).

### CODE REVIEW

**Review by knarayan — APPROVED** [review]
Had couple of minor comments, otherwise looks good.

File: tests/network-setups/corda/build.gradle [hunk 1]
```diff
@@ -100,9 +100,21 @@ task deployNodes(type: net.corda.plugins.Cordform) {
...
+            adminAddress("localhost:10049")
+        }
+        rpcUsers = [
+                [ user: "clientUser1", "password": "test", "permissions": ["ALL"]],
+                [ user: "driverUser1", "password": "test", "permissions": ["ALL"]]]
+    }
 }
 
 task prepareDockerNodes(type: net.corda.plugins.Dockerform, dependsOn: ['jar']) {
+    dockerImage = "corda/corda-zulu-java1.8-4.8"
```
[the remark this answers is not in the document]
Comment (sanvenDev) [reply tests/network-setups/corda/build.gradle:117]: 4.8 is the latest version?
It also has LTS support.

File: tests/network-setups/corda/constants.properties [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+platformVersion=10
 slf4jVersion=1.7.25
 nettyVersion=4.1.22.Final
 
 notaryRpcAddress=localhost:10003
 notaryAdminRpcAddress=localhost:10043
 partyARpcAddress=localhost:10006
-partyAAdminRpcAddress=localhost:10046
\ No newline at end of file
+partyAAdminRpcAddress=localhost:10046
```
[the remark this answers is not in the document]
Comment (sanvenDev) [reply tests/network-setups/corda/constants.properties:16]: Added.

**Comment by sanvenDev** [comment]
Fixes:
1. Typo in Stop networks in docs for setup with docker.
2. Fix `go.mod` in `interopcc` and `simpleassetandinterop`.
3. Fix Fabric-cli dockerfile.
4. Enabled go tests for github packages.

**Pull request merged**